main.tf

```diff
@@ -129,6 +128,7 @@ resource "aws_appautoscaling_policy" "autoscaling_read_replica_count" {
 }
 
 resource "aws_security_group" "this" {
+  count       = "${var.create_resources}"
```

Comment:
Lets keep empty lines after `count` and always have it as the first one because this is meta-argument.